backend/inspirehep/records/marshmallow/literature/latex.py

```diff
@@ -75,3 +78,33 @@ def get_collaborations(self, data):
             return missing
 
         return [collab["value"] for collab in data.get("collaborations")]
+
+    def get_note(self, data):
+
+        erratums = [
+            publication
+            for publication in get_value(data, "publication_info", [])
+            if publication.get("material") == "erratum"
+        ]
+
+        if not erratums:
+            return None
+
+        erratums_publication_infos = []
+        for erratum in erratums:
+            year = f"({erratum.get('year')})" if erratum.get("year") else ""
+            volume = erratum.get("journal_volume", "")
+            pages = LiteratureReader.get_page_artid_for_publication_info(erratum, "")
+            title = erratum.get("journal_title", "")
+            if title or volume:
+                title_volume_info = f"{title} {volume},".replace(" ,", ",")
+                info = f"{title_volume_info} {pages} {year}".strip(" ").strip(",")
+            elif pages or year:
+                info = f"{pages} {year}".strip(" ")
```

Comment:
Here you can use just `strip()` as by default it removes whitespaces https://docs.python.org/3.4/library/stdtypes.html?highlight=strip#str.strip